### DISCUSSION TITLE
[Feature] Past Date Flight Number Lookup

### DISCUSSION THREAD

**Discussion by amcmanu3** [johanohly/AirTrail#161] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi there! Super cool project here. Thank you for your work on that.

If at all possible it would be neat to try to add a date option when inputting a flight number when adding flights. Right now it looks like you're querying flights from "today". I'm not sure if this is a limitation of something upstream, but the import process could be better streamlined if users could lookup flight numbers that are in the past to have the flight info input automatically.

Thanks!

**Comment by johanohly** [comment]
Sorry for the late response, I must have overlooked this discussion.

The flight number lookup uses [this api](https://www.adsbdb.com/#href_callsign), which doesn't support specific dates. 

I have searched for an API that provides this capability for free for ages, and the best option I have found for now is the aforementioned.

**Comment by johanohly** [comment]
This has been implemented in v2.6.0, albeit with a little extra setup and a 3rd party account. This feature also takes into account the departure date you've entered. 
https://airtrail.johan.ohly.dk/docs/integrations/aerodatabox
